pkg/commands/render.go

```diff
@@ -48,7 +48,7 @@ func NewRenderCommand(commonOpts *CommonOptions) *cobra.Command {
 func renderManifests(cmd *cobra.Command, commonOpts *CommonOptions, opts *renderOptions, args []string) error {
 	var objs []runtime.Object
 
-	crd, err := manifests.LoadAPICRD()
+	crd, err := manifests.LoadAPICRD(manifests.CrdDefaultManifestPath)
```

Comment:
this is a good idea but we need to push it back. The fact we load the manifests is an implementation detail that the consumers of the `manifests` package should not concern with. I'm going to push a rename dropping the `Load` prefix to emphasize this: the `manifests` package will give the client the object, doesn't matter how it gets it.

Later, past MVP, we will add an option to possibly override the builtin manifests.

Reply:
So basically the client shouldn't be able to provide the path? 
If that's the case i need to remove also the path flag from the command

Reply:
yes, not now. We need to postpone this feature.